model/org.eclipse.birt.report.model/src/org/eclipse/birt/report/model/simpleapi/Style.java

```diff
@@ -72,27 +70,21 @@ public void setBackgroundAttachment(String value) throws SemanticException {
 		cmdStack.commit();
 	}
 
-	/*
-	 * (non-Javadoc)
+	/**
+	 * Get the background image source
 	 *
-	 * @see
-	 * org.eclipse.birt.report.engine.script.element.IStyleDesign#getBackgroundImage
-	 * ()
+	 * @return Return the background image source
 	 */
-
 	@Override
 	public String getBackgroundImage() {
 		return style.getBackgroundImage();
 	}
 
-	/*
-	 * (non-Javadoc)
+	/**
```

Comment:
actually I meant to remove the javadoc completely since it should be documented in the super